tools/ml-dataprep/runner.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Bootstrap module in charge of launching the extraction of the ML datasets
+from BigQuery."""
+
+
+import argparse
+import dataprep
+import metadata
+
+
+def initialise_parameters(args_parser):
+    """Define the arguments with the default values, parses the arguments
+    passed to the main program, and set the PARAMETERS global variable.
+
+    Args:     args_parser
+    """
+
```

Comment:
No blank lines after docstring (https://www.python.org/dev/peps/pep-0257/).

Reply:
Fixed in e3bdc093c431b17467a53f4d1e45957de9fdf1a9